.travis.yml

```diff
@@ -36,9 +36,6 @@ matrix:
       os: linux
       dist: trusty
       env: LDIST=DO_NOT_DEPLOY
-    - compiler: "gcc"
-      os: linux
-      dist: precise
```

Comment:
I think you could also remove `osx_image: xcode6.4`
you could do it in this commit, or make a new one ; doesn't matter much

Reply:
Added a new commit to remove the xcode 6.4 image